compiler/quilt/asa/img.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018, Quilt Data Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convenience functions for displaying images in Jupyter notebooks.
+
+`pip install quilt[img]`
+
+Or, in development:
+`pip install -e ./[img]`
+"""
+
+from math import ceil, floor, sqrt
+from six import string_types
+
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+
+from quilt.tools.const import ELLIPSIS
+from quilt.nodes import DataNode, GroupNode
+from quilt.tools.build import splitext_no_dot
+
+def plot(figsize=(10, 10), formats=None, limit=100, titlelen=10, **kwargs):
```

Comment:
A nicer way to do this would be `formats=lambda x: True` so that the value is always a callable and there's not a bunch of checking for `None` below.